server/lib/nl_page_config.py

```diff
@@ -341,37 +371,39 @@ def build_page_config(detection: Detection, data_spec: DataSpec,
       category.stat_var_spec[primary_sv].name = sv2name[primary_sv]
 
       # The per capita tile
-      tile = column.tiles.add()
-      sv_key = primary_sv + "_pc"
-      tile.stat_var_key.append(sv_key)
-      if classifier.type == ClassificationType.RANKING:
-        tile.type = subject_page_pb2.Tile.TileType.RANKING
-        if "CriminalActivities" in primary_sv:
-          # first check if "best" or "worst"
-          if RankingType.BEST in classifier.attributes.ranking_type:
-            tile.ranking_tile_spec.show_lowest = True
-          elif RankingType.WORST in classifier.attributes.ranking_type:
-            tile.ranking_tile_spec.show_highest = True
+      if _should_add_percapita(primary_sv):
+        tile = column.tiles.add()
+        sv_key = primary_sv + "_pc"
+        tile.stat_var_key.append(sv_key)
+        if classifier.type == ClassificationType.RANKING:
```

Comment:
I think moving `if classifier.type == ClassificationType.RANKING:` under `if _should_add_percapita(primary_sv):` means we won't ever show ranking charts for the weather data. Is this intended behavior?

Reply:
not for per capita. there's another block above for non-percapita

Reply:
gotcha. thanks for clarifying!